.github/workflows/release-prod.yml

```diff
@@ -4,7 +4,7 @@ run-name: Production release PR
 on:
   push:
     branches:
-      - main-v4
+      - main
```

Comment:
We could/should add `release/v4` here, too?

At least Stately did it: https://github.com/statelyai/xstate/pull/878/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34

Reply:
I will do it but it does not really matter because the Github actions are executed per branch. If you check the `main-v4` you will see that this line is updated with the `main-v4`.
If we update this, we could avoid introducing a specific commit to each release branch.

Reply:
I think yes

Reply:
done!